common/lib/xmodule/xmodule/templates/problem/string_response.yaml

```diff
@@ -18,26 +18,21 @@ metadata:
         Nanjing Higher Normal Institute first admitted female students in 1920.
         [explanation]
 
-data: |
-    <problem>
-    <p>In text input problems, also known as "fill-in-the-blank" problems, 
-    learners enter text into a response field. The text that the learner enters 
-    must match your specified answer text exactly. You can specify more than 
-    one correct answer. Learners must enter a response that matches one of the 
-    correct answers exactly.</p>
-    <p>When you add the problem, be sure to select <strong>Settings</strong> 
-    to specify a <strong>Display Name</strong> and other values that apply.</p>
-    <p> You can use the following example problem as a model.</p>
-      <p>What was the first post-secondary school in China to allow both male and female students?</p>
-      <stringresponse answer="Nanjing Higher Normal Institute" type="ci" >
-        <additional_answer>National Central University</additional_answer>
-        <additional_answer>Nanjing University</additional_answer>
-        <textline label="What was the first post-secondary school in China to allow both male and female students?" size="40"/>
-      </stringresponse>
-    <solution>
-      <div class="detailed-solution">
-        <p>Explanation</p>
-        <p>Nanjing Higher Normal Institute first admitted female students in 1920.</p>
-      </div>
-    </solution>
-    </problem>
+data:   |
+        <problem>
+            <stringresponse answer="Nanjing Higher Normal Institute" type="ci">
+                <p>In text input problems, also known as "fill-in-the-blank" problems, learners enter text into a response field. The text that the learner enters must match your specified answer text exactly. You can specify more than one correct answer. Learners must enter a response that matches one of the correct answers exactly.</p>
+                <p>When you add the problem, be sure to select <strong>Settings</strong> to specify a <strong>Display Name</strong> and other values that apply.</p>
```

Comment:
hello @muhammad-ammar here is my suggestion. The guidance about the templates on the wiki page (https://openedx.atlassian.net/wiki/pages/viewpage.action?pageId=30966057) is very helpful. Let's simplify these templates a lot!
Rather than try to draft them in PR comments, I will add text to the child wiki pages (like https://openedx.atlassian.net/wiki/display/A11Y/Dropdown) and invite comment there.